app/core/config.py

```
@@ -0,0 +1,37 @@
+    
+    # MongoDB 설정
+    # MONGODB_URL: Optional[str] = None
+
+    model_config = SettingsConfigDict(
+        env_file=".env",        # .env 파일을 읽어옴
+        case_sensitive=True,    # 대소문자 구분
+        extra="ignore"         # 정의되지 않은 환경변수가 있어도 에러내지 않고 무시
+    )
+
+# 싱클톤 패턴
+@lru_cache()
```

Ruff (0.14.11)

Unnecessary parentheses to `functools.lru_cache`

Remove unnecessary parentheses

(UP011)